ykrt/src/compile/jitc_yk/codegen/x64/mod.rs

```diff
@@ -121,6 +121,8 @@ impl CodeGen for X64CodeGen {
         m: Module,
         mt: Arc<MT>,
         hl: Arc<Mutex<HotLocation>>,
+        // Stack pointer offset from the base pointer of interpreter frame as defined in
+        // [YkSideTraceInfo].
```

Comment:
I think this should be `[YkSideTraceInfo::sp_offset]` or similar.

Reply:
Here you go: [293a9b5](https://github.com/ykjit/yk/pull/1422/commits/293a9b518a5cf75c82c5abeaf33b9dacc1febc13)